python/ray/_private/ray_constants.py

```diff
@@ -162,6 +162,7 @@ def env_set_by_user(key):
 DEFAULT_DASHBOARD_PORT = 8265
 DASHBOARD_ADDRESS = "dashboard"
 DASHBOARD_CLIENT_MAX_SIZE = 100 * 1024**2
+DASHBOARD_START_TIMEOUT_S = env_integer("RAY_DASHBOARD_START_TIMEOUT_S", 60)
```

Comment:
Does `env_integer` enforce a positive value? If not, should we check that this is always positive?